docs/envvars.rst

```diff
@@ -55,10 +76,11 @@ applicable.
       - Indentation string for multiline input
     * - MOUSE_SUPPORT
       - ``False``
-      - Enable mouse support in the prompt_toolkit shell. This allows clicking
-        for positioning the cursor or selecting a completion. In some terminals
-        however, this disables the ability to scroll back through the history
-        of the terminal.
+      - Enable mouse support in the prompt_toolkit shell.
+        
+        This allows clicking for positioning the cursor or selecting a completion. In some terminals
+        however, this disables the ability to scroll back through the history of the terminal.
+        
```

Comment:
At least on the Github viewer, without those new blank newlines the bullet points and line breaks are completely ignored and everything is shoved into one single, soft-wrapped line. I thought it is much more readable with the newlines.